docs/tutorial/creating_targets.rst

```diff
@@ -43,7 +43,7 @@ by running the following commands:
 
 .. code-block:: bash
 
-    cd <BLT repository/docs/tutorial/calc_pi
+    cd <BLT repository>/docs/tutorial/calc_pi
     mkdir build
     cd build
     cmake -DBLT_SOURCE_DIR=../../.. ..
```

Comment:
We should add a link to the API docs for `blt_add_executable`  (and `blt_add_library`), either in the text or in the `hint` admonition.

E.g. something like:
```
The full list of supported parameters can be found here: https://llnl-blt.readthedocs.io/en/latest/api/target.html#blt-add-executable
```

Reply:
Even better: Could we add a `collapsible` toggle to show this inline?